packages/clerk-js/src/ui/hooks/useClerkModalStateParams.tsx

```diff
@@ -4,23 +4,23 @@ import { CLERK_MODAL_STATE } from '../../core/constants';
 import { readStateParam, removeClerkQueryParam } from '../../utils';
 
 export const useClerkModalStateParams = () => {
-  const contentRef = React.useRef({ startPath: '', path: '', componentName: '', socialProvider: '' });
+  const [state, setState] = React.useState({ startPath: '', path: '', componentName: '', socialProvider: '' });
```

Comment:
❓Should we keep the naming `content` ?
```suggestion
  const [content, setContent] = React.useState({ startPath: '', path: '', componentName: '', socialProvider: '' });
```

Reply:
@panteliselef It doesn’t really matter in this case since we don’t export it and use it directly 🙃